### PR TITLE
Updating CORS for the Frontend Review Dashboard

### PR DESCRIPTION
**Pull request by timwright12**
Trying to update the CORS policy for api.va.gov/v0/backend_statuses to include department-of-veterans-affairs.github.io

Need some good eyes on this though because I really don't know Ruby and I don't want to break anything

Original ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/177

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ vsp_environment:
 virtual_hosts: ["127.0.0.1", "localhost"] # Safe host names
 
 # For CORS requests; separate multiple origins with a comma
-web_origin: http://localhost:3000,http://localhost:3001,http://127.0.0.1:3000,http://127.0.0.1:3001,null
+web_origin: http://localhost:3000,http://localhost:3001,http://127.0.0.1:3000,http://127.0.0.1:3001,https://department-of-veterans-affairs.github.io,null
```
Comment (LindseySaari) [inline config/settings.yml:8]: Hey @timwright12, I think you will need to update this setting in the devops repo. The settings.yml gets overwritten on the deploy with the settings here for each environment: [prod](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/prod-settings.local.yml.j2), [staging](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/staging-settings.local.yml.j2), [sandbox](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/sandbox-settings.local.yml.j2), [dev](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/dev-settings.local.yml.j2)

Comment (timwright12) [reply config/settings.yml:8]: @LindseySaari awesome thank you! Do I need to merge this PR and one in devops or just devops?

Comment (LindseySaari) [reply config/settings.yml:8]: I believe just devops unless you need that change for local development (which it doesn't appear that you do)

Comment (timwright12) [reply config/settings.yml:8]: excellent, thank you. I'll close this out

**Pull request closed without merging**